src/assets/socis.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+function SOCIS({ width, height, viewbox }) {
```

Comment:
All the other logos logos are called `<club>Logo`  and the socis one is called `SOCIS` this should be changed to be constant

Reply:
Fixed, they should all be consistently named now